services/catalog/src/simcore_service_catalog/services/director.py

```diff
@@ -113,18 +117,18 @@ def __init__(self, base_url: str, app: FastAPI):
     async def close(self):
         await self.client.aclose()
 
-    # OPERATIONS
-    # TODO: policy to retry if NetworkError/timeout?
-    # TODO: add ping to healthcheck
+    #
+    # Low level API
+    #
 
-    @_safe_request
+    @_return_data_or_raise_error
     async def get(self, path: str) -> httpx.Response:
         # temp solution: default timeout increased to 20"
```

Comment:
does not look so temporary anymore.

Reply:
@sanderegg AFAIK these comments were added by @odeimaiz based on some heuristics he determined. The timeout is still 20'' so the comment still applies. 
One of the reasons to increase the test coverage is to determine whether with the new approach we can drop some of these heuristics. In the meantime I rather leave them as they are.